horizon/manifest.jsonnet

```diff
@@ -27,7 +27,7 @@ kpm.package({
 
     network: {
       external_ips: [],
-      port: 80,
+      port: 8080,
```

Comment:
Apache2 has extended linux_capabilities about cap_net_bind_service. So you don't need to do it even if you start as a non-root.